CHANGELOG.md

```diff
@@ -81,6 +81,10 @@ IMPROVEMENTS:
   pagination [#378](https://github.com/vmware/go-vcloud-director/pull/378)
 * Improved LDAP container related tests to use correct port mapping for latest LDAP container version 
   [#378](https://github.com/vmware/go-vcloud-director/pull/378)
+* Added support for Shareable disks, i.e., independent disks that can be attached to multiple VMs which is available from
```

Comment:
Could you move this changelog change to `.changes/v2.15.0` In this folder, we create the file with PR number and store the changelog message.

Reply:
done @vbauzysvmware . please take a look at the second commit. (I had to force-push since my branch was quite outdated)

Reply:
thank you @arunmk